docs/api/selector.md

```diff
@@ -34,39 +46,26 @@ enzyme also gives support for the following contextual selectors
 .foo input
 ```
 
-**Want more CSS support?**
-
-PRs implementing more support for CSS selectors will be accepted and is an area of development for
-enzyme that will likely be focused on in the future.
-
-
+**The Key and Ref Props**
 
-### 2. Prop Selector
```

Comment:
Why remove this section?

Reply:
it's part of regular CSS so I merged it into the CSS case.  Isn't it?

Reply:
yeah, but the result looks almost the same.  '[num=51]' will match num={51}, but not num='51'.  And, '[num="51"]' will match num='51' but not num={51}.  I think that should get a better description.
Meanwhile, these constructs can mix in with the rest of the CSS selector, so isn't it a part of CSS?

Reply:
It would be good to have a full accounting of the differences between enzyme selectors and css selectors.  There was one phrase in there, "Further, enzyme supports combining any of those supported syntaxes together to uniquely identify a single node."  I misinterpreted that to mean you can only select one node if you combine syntaxes.  I rewrote the section.